src/components/RenderRouter/ContentItem.scss

```diff
@@ -313,7 +313,7 @@
         position: static;
         padding-bottom: 5vw;
         // padding-top: 45vw;
-        display: inline-block;
+        display: relative;
```

Comment:
This was causing the footer and partialHero to conflict...probably best to use relative for the main frames of the components...